addons/civ_placement/fnc_CP.sqf

```diff
@@ -212,8 +212,8 @@ switch(_operation) do {
 
         if (isnil "_args") exitwith {_result = DEFAULT_WITH_PLACEMENT};
 
-        if (typename _args == "STRING") then {_args = call compile _args};
-        if (typename _args == "BOOL") then {_args = _args};
+        if (_args isEqualType "") then {_args = (missionNamespace getVariable [_args,[]])};
```

Comment:
You don't know what's in there. According to line 216 it expects a boolean. And if the string contains "true" then this won't work.
Line 210 also sounds like it would be a boolean.

Reply:
As mentioned on discord, `_args` can only be one of `"true"`, `"false"`, `true` and `false`. So it should probably become `if (_args isEqualType "") then {_args = (_args == "true")};`.